### ISSUE TITLE
ITV Shows won't convert and upload to itunes

### ISSUE THREAD

**Issue by pmhird** [GetiPlayerAutomator/get-iplayer-automator#449]
After successfully downloading a show from ITV, the conversion to MP4 stalled after 48kb. The line in automator reads "Download complete" rather than the "Complete and in iTunes". The .partial.flv file found in the download folder was complete, and I successfully converted it to iTunes format using a third party application. BBC programmes seem unaffected however.

Using 1.3.7.1 on 10.6.8

Thanks

**Comment by pmhird** [comment]
Well its not working for me. Show downloads just fine, and then it leaves me with a .partial.flv that is complete, and a .mp4 file that is just a few kb. BBC radio and TV working just fine. I first noticed it with Downton Abbey Christmas special but it also did it for the movie Santa Baby. Unfortunately I wasn't in verbose mode so didn't catch why. I tried a couple of quick kids shows to see what would happen with a shorter file and same story. I'm still getting "download complete" instead of "complete and in iTunes" 

Here's the last part of the log that deals with the conversion...

INFO: Converting FLV File to MP4
FFmpeg version git-44002d8, Copyright (c) 2000-2011 the FFmpeg developers
  built on Jan 25 2011 21:13:27 with gcc 4.2.1 (Apple Inc. build 5664)
  configuration: --prefix=/Volumes/Ramdisk/sw --enable-gpl --enable-pthreads --enable-version3 --enable-nonfree --enable-libspeex --enable-libvpx --disable-decoder=libvpx --enable-libfaac --enable-libmp3lame --enable-libtheora --enable-libvorbis --enable-libx264 --enable-avfilter --enable-libopencore_amrwb --enable-libopencore_amrnb --enable-filters --arch=x86 --enable-runtime-cpudetect
  libavutil    50. 36. 0 / 50. 36. 0
  libavcore     0. 16. 1 /  0. 16. 1
  libavcodec   52.108. 0 / 52.108. 0
  libavformat  52. 94. 0 / 52. 94. 0
  libavdevice  52.  2. 3 / 52.  2. 3
  libavfilter   1. 74. 0 /  1. 74. 0
  libswscale    0. 12. 0 /  0. 12. 0
INFO: Finished Converting.
INFO: Exit Code = 1

AppController: Downloads Finished
Adding Horrid Henry  - Horrid Henry and the Dangerous Data to iTunes
Can't add to iTunes; incompatible format.
			iTunes Compatible Modes: Flash - High, Flash - Standard, Flash - HD, iPhone, Radio - MP3, Podcast
